templates/bot/ts/workflow/package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "command-response-bot",
```

Comment:
workflow-bot?

Reply:
Good catch, the description should be updated as well